guide/src/reference/attributes/on-rust-exports/constructor.md

```diff
@@ -32,3 +32,41 @@ import { Foo } from './my_module';
 const f = new Foo();
 console.log(f.get_contents());
 ```
+
+## Caveats
+
+Starting from v0.2.48 there is a bug in the `wasm-bindgen-cli-support` crate which breaks inheritance of exported Rust structs from JavaScript side (see [#3213](https://github.com/rustwasm/wasm-bindgen/issues/3213)). If you want to inherit a Rust struct such as:
```

Comment:
I don't think anyone cares about which internal crate caused the problem; you should probably just say `wasm-bindgen` or 'the `wasm-bindgen` CLI'.

Reply:
I wasn't sure if it was worth even including this. But yeah good point, I'll make these changes.